.github/workflows/feg-workflow.yml

```diff
@@ -0,0 +1,70 @@
+name: feg-workflow
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches: [ master ]
+
+jobs:
+
+########## FEG-PRECOMMIT ##########
+
+  feg-precommit:
+
+    runs-on: ubuntu-latest
+
+    env:
+      GO111MODULE: on
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Export Magma root path
+      run: echo "MAGMA_ROOT=${PWD}" >> $GITHUB_ENV
+    - name: Run golang_before_install.sh script
+      run: ./circleci/golang_before_install.sh
+    - name: Run go mod download with retry
+      uses: nick-invision/retry@v2
+      with:
+        command: cd ${MAGMA_ROOT}/cwf/gateway && go mod download
+        timeout_minutes: 10
+    - name: make feg precommit
+      run: |
+          cd ${MAGMA_ROOT}/feg/gateway
+          make -C ${MAGMA_ROOT}/feg/gateway precommit
+#    - magma_slack_notify
+
+########## FEG-BUILD ##########
+
+  feg-build:
```

Comment:
note this flow will be changes in this PR on Circle CI

https://github.com/magma/magma/pull/6288/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47R715

Reply:
Thanks for this. Let me update my PR